server/routes/auth.js

```diff
@@ -139,6 +139,12 @@ router.get('/logout', (req, res) => {
   res.clearCookie('jwt').send();
 });
 
+router.use('/users', (req, res, next) => {
+  if (req.user && req.user.isAdmin) {
+    next();
+  }
```

Comment:
We should respond with status code `403`. I didn't test this but this should work:

```js
router.use('/users', (req, res, next) => {
  if (req.user && req.user.isAdmin) {
    next();
    return;
  }

  res.status(401).send('Must be admin');
});
```